src/test/java/Homework19.java

```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.WebElement;
+
+    public class Homework19 extends BaseTest {
+
+@Test
+public void deletePlaylist() throws InterruptedException {
+    setupClass();
+
+    navigateToPage();
+    provideEmail("test@test.com");
+    providePassword("te$t$tudent");
+    clickSubmit();
+    choosePlaylist();
+    clickDeletePlaylistBtn();
+    Assert.assertTrue(confirmNotification().contains("Deleted playlist"));
+}
+
+
+        public void choosePlaylist() throws InterruptedException {
+    WebElement clickPlaylist = driver.findElement(By.xpath("//*[@id='playlists']/ul/li[3]/a"));
+    clickPlaylist.click();
+    Thread.sleep(1000);
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)